Gemfile

```diff
@@ -2,25 +2,22 @@
 
 source 'https://rubygems.org'
 
+# stanford dlss specific gems
 gem 'assembly-objectfile'
-gem 'config', '~> 3.1'
 gem 'dor-services-client', '~> 12.0'
 gem 'dor-workflow-client', '~> 4.0'
 gem 'druid-tools'
-gem 'fastimage', '~> 2.2'
-gem 'ffi-geos', '~> 1.0', require: false # XXX: where is this used?
```

Comment:
ffi-geos gem was added by [PR](https://github.com/sul-dlss/gis-robot-suite/commit/738d7bcc767b386e24e4da831a86d728d04dc20d) and never used